addons/mail/static/tests/tours/mail_composer_test_tour.js

```diff
@@ -150,5 +150,34 @@ registry.category("web_tour.tours").add("mail/static/tests/tours/mail_composer_t
                 }
             },
         },
+        // Test that different lines are in different p elements in editor
+        {
+            content: "Write first line",
+            trigger: ".o-mail-Composer-input",
+            run: "edit abc",
+        },
+        {
+            content: "Press enter to go to next line",
+            trigger: ".o-mail-Composer-input",
+            run: "press enter",
+        },
+        {
+            content: "write second line",
+            trigger: ".o-mail-Composer-input",
+            run: "fill efg",
+        },
+        {
+            content: "Open full composer",
+            trigger: "button[aria-label='Full composer']",
+            run: "click",
```

Comment:
Note for forward-port: the `aria-label` has been replaced by `title`, and value has been changed to "Open Full Composer".